sites/azure.ouroboros.talk.conf

```diff
@@ -0,0 +1,40 @@
+# Static Ouroboros talk pages migrated from s3 to azure blob store
+# but have invalid file names for azure (ending in .)
+#
+# These files were renamed to have a valid end char '_'
+#
+# we now redirect any URLs that end in '.' char
+# to the moved files that end in '_'
+server {
+    include /etc/nginx/ssl.default.conf;
+    # all domains that had invalid file names
+    server_name radiotalk.galaxyzoo.org
+                talk.chicagowildlifewatch.org
+                talk.diskdetective.org
+                talk.galaxyzoo.org
+                talk.planethunters.org
+                talk.snapshotserengeti.org
+                talk.wormwatchlab.org;
+
+  # for /users/ static pages that end in '.' char
+  # '.' chars are illegal in azure so we mapped it to a
+  # '_' char path prefix in azure blob store
+  # we will proxy pass to the azure blob mapped '_' file prefix
+  location ~* ^(/users/.+)\.(\/.*)?$ {
```

Comment:
Are we sure this only affects users and not tags or anything else?

Reply:
I'm not sure - i'll progress as is and look to resync after renaming the known problem files (bash script) and will revisit any other problematic paths when we find them.